bin/dartdoc.dart

```diff
@@ -249,104 +255,108 @@ main(List<String> arguments) async {
 
 ArgParser _createArgsParser() {
   var parser = new ArgParser();
-  parser.addFlag('help',
-      abbr: 'h', negatable: false, help: 'Show command help.');
-  parser.addFlag('version',
-      help: 'Display the version for $name.', negatable: false);
   parser.addFlag('add-crossdart',
       help: 'Add Crossdart links to the source code pieces.',
       negatable: false,
       defaultsTo: false);
-  parser.addFlag('sdk-docs',
-      help: 'Generate ONLY the docs for the Dart SDK.', negatable: false);
-  parser.addFlag('show-warnings',
-      help: 'Display warnings.', negatable: false, defaultsTo: false);
-  parser.addFlag('show-progress',
-      help: 'Display progress indications to console stdout', negatable: false);
-  parser.addOption('sdk-readme',
-      help: 'Path to the SDK description file.  Deprecated (ignored)');
-  parser.addOption('input', help: 'Path to source directory.');
-  parser.addOption('output',
-      help: 'Path to output directory.', defaultsTo: defaultOutDir);
-  parser.addMultiOption('header',
-      splitCommas: true, help: 'paths to header files containing HTML text.');
+  parser.addOption('ambiguous-reexport-scorer-min-confidence',
+      help:
+          'Minimum scorer confidence to suppress warning on ambiguous reexport.',
+      defaultsTo: "0.1",
```

Comment:
Perhaps stick with single-quotes for consistency?

Reply:
done